docs/contributing.rst

```diff
@@ -26,7 +26,12 @@ See the project's `Makefile` targets for a list of common developer tasks, which
 Notes on the Test Suite
 -----------------------
 
-To test all the API endpoints, the account that you use for testing must be an *admin* user for your Webex Organization.  Additionally, you should know that that the testing process creates some test people, rooms, messages, teams, and etc. as part of executing the test suite. We strongly recommend *NOT* running the test suite using your personal Webex account (not that you can't; it's just that you probably don't want it cluttering your account with all these test artifacts).
+To test all the API endpoints, the account that you use for testing must be an *admin* and *compliance officer* user for your Webex Organization.  Additionally, you should know that that the testing process creates some test people, rooms, messages, teams, and etc. as part of executing the test suite. 
+
+We strongly recommend *NOT* running the test suite using your personal Webex account (not that you can't; it's just that you probably don't want it cluttering your account with all these test artifacts).
+
+Webex now offers a free developer sandbox organization that you can use for testing purposes.  You can request the sandbox at https://developer.webex.com/docs/developer-sandbox-guide.
+Once you have your sandbox organization, you can create a test account with *admin* and *compliance officer* privileges via [Webex Control Hub](https://admin.webex.com) and use that account for testing. (Be sure to login to Control Hub with the new admin so that the roles are assigned properly to the Webex token.)
```

Comment:
_A minor formatting consistency nit:_ If we intend lines 33 and 34 to be in the same paragraph, we should combine these lines for consistency with the rest of this doc. If we intend them to be separate paragraphs, then there should be a blank line between 33 and 34. 🙂